examples/src/main/java/org/apache/ignite/examples/springdata/SpringDataExample.java

```diff
@@ -131,7 +135,7 @@ private static void findPersons() {
      * Execute advanced queries over the repository.
      */
     private static void queryRepository() {
-        System.out.println("\n>>> Persons with name 'John':");
+        System.out.println("\n>>> Persons with name 'John' and 'Brad':");
```

Comment:
Why did you update this value? It does not seem to reflect what happens below.

Reply:
Mistake, reverted.